src/tsconfig.json

```diff
@@ -1,12 +1,16 @@
 {
     "compilerOptions": {
+        "target": "es5",
         "noImplicitAny": true,
         "noImplicitReturns": true,
         "noImplicitThis": false,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
         "suppressImplicitAnyIndexErrors": true,
         "strictNullChecks": false,
+        "lib": [
+            "dom", "es2015" // dom for console.log only
```

Comment:
Probably `["es5", "es2015.core"]` with this declaration 

`declare var console: {log(...args: any[]): void}`